docs/jssdk.md

```diff
@@ -210,11 +217,11 @@ To create the 'My Account' menu for a logged in customer, you need the following
 
```

Comment:
checkAccessForAsset method accepts only one parameter asset id. You need to change the upper code block with this:
```javascript
InPlayer.Asset
.checkAccessForAsset(ASSET_ID)
.then(data => console.log(data))
.catch(error => error.response.json().then(data => console.log("Error", data)));
```

Reply:
Done.